.travis.yml

```diff
@@ -1,20 +1,22 @@
 language: java
-install: skip
+install: 
+  - git clone https://github.com/sef-global/scholarx-frontend
+  - cd scholarx-frontend
+  - git checkout deploy
+  - npm install
+  - npm run build
+  - cd ../
 before_script:
-  - bash scripts/setup.sh
+  - cp scholarx-frontend/dist/bundle.js src/main/resources/static/dist/
+  - sudo rm -R scholarx-frontend
```

Comment:
Is it alright to add only the bundle.js file??
Don't we need to include the image files and an index.html file for this to work??

Reply:
Yeah, you are correct. I'll add the other files

Reply:
Updated